core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -169,11 +171,16 @@ public SELF withServices(@NonNull String... services) {
     }
 
     private void createServices() {
+        // Apply scaling
+        final String servicesWithScalingSettings = Stream.concat(services.stream(), scalingPreferences.keySet().stream())
```

Comment:
Scaling behaviour changed; rather than old behaviour of:
```
docker-compose scale someservice=2
docker-compose up -d
```
we now have to do:
```
docker-compose up -d --scale someservice=2
```